webcompat/static/css/src/issue-wizard.css

```diff
@@ -1027,6 +1032,7 @@
     font-size: 90%;
     font-weight: 600;
     padding: 11px 36px 11px 8px;
+    transition: box-shadow .3s;
```

Comment:
This should be moved out of `@media (min-width: 665px)` as well in order to work for both mobile and desktop
```css
.issue-form .text-field {
    transition: box-shadow .3s;
}
```